### PR TITLE
Issue #1625: Display age of diagnosis for cancers

### PR DESCRIPTION
**Pull request by veronikaslc**
Fixed.

### CODE REVIEW

**Comment by veronikaslc** [comment]
@JKereliuk , @DeanWay please, review :)

**Comment by DeanWay** [comment]
Date of diagnosis for a cancer should not be shown if the node is marked "unaffected" for that cancer

File: components/pedigree/resources/src/main/resources/pedigree/view/personVisuals.js [hunk 1]
```diff
@@ -626,6 +627,65 @@ var PersonVisuals = Class.create(AbstractPersonVisuals, {
         }
         this.drawLabels();
     },
+    
+    
+    /**
+     * Returns this Person's cancer age of onset labels
+     *
+     * @method getCommentsLabel
```
Comment (sdumitriu) [inline components/pedigree/resources/src/main/resources/pedigree/view/personVisuals.js]: Wrong documentation.

File: components/pedigree/resources/src/main/resources/pedigree/view/personVisuals.js [hunk 2]
```diff
@@ -626,6 +627,65 @@ var PersonVisuals = Class.create(AbstractPersonVisuals, {
...
+                	}
+                	this.getCancerAgeOfOnsetLables()[cancerName] && this.getCancerAgeOfOnsetLables()[cancerName].remove();
+                	this._cancerAgeOfOnsetLables[cancerName] = editor.getPaper().text(this.getX(), this.getY(), text).attr(PedigreeEditor.attributes.cancerAgeOfOnsetLables);
+                	this._cancerAgeOfOnsetLables[cancerName].node.setAttribute("class", "field-no-user-select");
+                    this._cancerAgeOfOnsetLables[cancerName].alignTop = true;
+                    this._cancerAgeOfOnsetLables[cancerName].addGap   = true;
+                }
+            }
+    		
+    	} else {
```
Comment (sdumitriu) [inline components/pedigree/resources/src/main/resources/pedigree/view/personVisuals.js]: Don't use tabs for indentation.

**Pull request merged**